WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -3,68 +3,105 @@ import SwiftUI
                     .lineLimit(1)
                     .font(.headline)
                     .foregroundStyle(.primary)
 
-                if !author.isEmpty {
-                    Text(Strings.author(author))
-                        .lineLimit(1)
-                        .font(.caption)
-                        .foregroundStyle(.secondary)
-                }
-
                 Group {
-                    if shortDescription.isEmpty {
+                    if plugin.shortDescription.isEmpty {
                         Text(Strings.noDescriptionAvailable)
                             .font(.system(size: descriptionFontSize).italic())
                     } else if let html = renderedDescription() {
                         Text(html)
                     } else {
-                        Text(shortDescription)
+                        Text(plugin.shortDescription)
                             .font(.system(size: descriptionFontSize))
                     }
                 }
+                .lineLimit(2)
                 .padding(.vertical, 4)
 
-                Text(Strings.version(version))
+                Text(Strings.version(plugin.version))
                     .font(.caption)
                     .foregroundStyle(.secondary)
             }
+
+            Spacer()
+        }
+        .overlay(alignment: .bottomTrailing) {
+            Menu {
+                Section {
+                    if plugin.isActive {
+                        Button(Strings.deactivate, systemImage: "bolt.slash") {
+                            Task {
+                                await viewModel.toggle(slug: plugin.slug)
+                            }
+                        }
+                    } else {
+                        Button(Strings.activate, systemImage: "bolt") {
+                            Task {
+                                await viewModel.toggle(slug: plugin.slug)
+                            }
+                        }
+                        Button(Strings.delete, systemImage: "trash") {
+                            Task {
+                                await viewModel.uninstall(slug: plugin.slug)
+                            }
+                        }
+                    }
+                }
+                .disabled(isUpdating)
+
+                if wpOrgURL != nil {
+                    Section {
+                        Button {
+                            showingSafariView = true
+                        } label: {
+                            Label(Strings.viewOnWordPressOrg, systemImage: "safari")
+                        }
+                    }
+                }
+            } label: {
+                Image(systemName: "ellipsis")
+                    .padding(4)
+                    .frame(width: 44, height: 44, alignment: .bottomTrailing)
+                    .contentShape(Rectangle())
+            }
+            .foregroundStyle(.secondary)
+        }
+        .sheet(isPresented: $showingSafariView) {
```

Comment:
I'd delegate it to the top view and use `.sheet(item:` just to be safe. It's fairly easy for `PluginListItemView` to get reloaded and lose its identity and state. It should probably be OK in this case.